skin.css

```diff
@@ -2,12 +2,65 @@
     background-color: #ddd;
 }
 
-button.btn {
+.btn {
+  width:170px
   border-radius: 3px;
-  border-style: solid;
-  border-width: 1px;
+  background-color:white;
   font-weight: normal;
-  line-height: 1.15;
-  text-align: center;
-  vertical-align: middle;
+  font-family:sans-serif;
+  padding: 15px;
+  border: 1px; solid lightgrey
+
+
+}
+
+
+  .btn--default{
+   border-style:lightgrey
+    
+       } 
+.btn--primary{
+  background-color:blue;
+  color:white;
+  }
+  .btn--secondary{
+    background-color:gainsboro;
+    color:grey;
+    
+  }
+  .btn--large{
+    width:250px;
+    padding:20px;
+    color:blue;
+  }
+  .btn--extra-large{
+    width:350px;
+    padding:20px;
+    background-color: white;
+    color: blue;
+    font-size:20px;
+
+  }
+  nav.ul{
+    float: left;
+    
+  }
+  ul li{
+    float: left;
+    padding:20px;
+    margin: 20px;
+    width: 80px;
+    list-style-type: none;
+    border: 1px dotted grey;
+    text-align: center;
+}
+  li a{
+    transition-delay: 0.5s;
+    text-decoration: none;
+ }
+ :hover{

```

Comment:
Writing only `:hover` is the same as `html:hover`. Similarly, `:first-child` is resolved to `html:first-child`. This is why your entire page has a yellow background colour.

In order to apply these selectors only to your `li` element, you need to do `li:hover` and `li:first-child`. Everything will look nicer immediately. :)

If using normal CSS:

```
li:first-child {
    background-color: yellow;
}
```

If using SASS:

```
li {
  &:first-child {
    background-color: yellow;
  }
}
```

[More reading on pseudo-classes (hover, first-child, etc) >](https://css-tricks.com/pseudo-class-selectors/)